service/src/main/java/uk/gov/hmcts/reform/fpl/service/casesubmission/CaseSubmissionTemplateDataGenerationService.java

```diff
@@ -0,0 +1,742 @@
+                stringBuilder.append(orders.getEmergencyProtectionOrderDirections().stream()
+                    .map(EmergencyProtectionOrderDirectionsType::getLabel)
+                    .collect(joining(NEW_LINE)));
+            }
+
+            stringBuilder.append(NEW_LINE);
+            appendEmergencyProtectionOrderDirectionDetails(orders, stringBuilder);
+            appendDirectionsAndDirectionDetails(orders, stringBuilder);
+        }
+
+        return StringUtils.isNotEmpty(stringBuilder.toString()) ? stringBuilder.toString().trim() : DEFAULT_STRING;
+    }
+
+    private void appendEmergencyProtectionOrderDirectionDetails(final Orders orders,
+                                                                final StringBuilder stringBuilder) {
+        if (StringUtils.isNotEmpty(orders.getEmergencyProtectionOrderDirectionDetails())) {
+            stringBuilder.append(orders.getEmergencyProtectionOrderDirectionDetails());
+            stringBuilder.append(NEW_LINE);
+        }
+    }
+
+    private void appendDirectionsAndDirectionDetails(final Orders orders, final StringBuilder stringBuilder) {
+        if (StringUtils.isNotEmpty(orders.getDirections())) {
+            stringBuilder.append(orders.getDirections());
+            stringBuilder.append(NEW_LINE);
+        }
+
+        if (StringUtils.isNotEmpty(orders.getDirectionDetails())) {
+            stringBuilder.append(orders.getDirectionDetails());
+        }
+    }
+
+    private String getGroundsForEPOReason(final List<OrderType> orderTypes, final GroundsForEPO groundsForEPO) {
+        if (isNotEmpty(orderTypes) && orderTypes.contains(OrderType.EMERGENCY_PROTECTION_ORDER)) {
+
+            if (isNotEmpty(groundsForEPO) && isNotEmpty(groundsForEPO.getReason())) {
+                return groundsForEPO.getReason()
+                    .stream()
+                    .map(reason -> EmergencyProtectionOrderReasonsType.valueOf(reason).getLabel())
+                    .collect(joining(NEW_LINE + NEW_LINE));
+            }
+
+            return DEFAULT_STRING;
+        }
+
+        return EMPTY;
+    }
+
+    private String getThresholdDetails(final Grounds grounds) {
+        return (isNotEmpty(grounds) && StringUtils.isNotEmpty(grounds.getThresholdDetails()))
+            ? grounds.getThresholdDetails() : DEFAULT_STRING;
+    }
+
+    private String buildGroundsThresholdReason(final Grounds grounds) {
+        StringBuilder stringBuilder = new StringBuilder();
+        if (isNotEmpty(grounds) && isNotEmpty(grounds.getThresholdReason())) {
+            grounds.getThresholdReason().forEach(thresholdReason -> {
+                if (StringUtils.equals(thresholdReason, "noCare")) {
+                    stringBuilder.append("Not receiving care that would be reasonably expected from a parent.");
+                    stringBuilder.append(NEW_LINE);
+
+                } else if (StringUtils.equals(thresholdReason, "beyondControl")) {
```

Comment:
Any reason this isn't just
```suggestion
                } else if ("beyondControl".equals(thresholdReason)) {
```

Reply:
Same for the one above